hummingbot/core/data_type/order_book.pyx

```diff
@@ -328,6 +328,33 @@ cdef class OrderBook(PubSub):
 
         return OrderBookQueryResult(NaN, volume, result_vwap, min(total_volume, volume))
 
+    cdef OrderBookQueryResult c_get_vwap_for_quote_volume(self, bint is_buy, double quote_volume):
+        cdef:
+            double total_cost  = 0
+            double total_volume = 0
+            double result_vwap = NaN
+
+        if is_buy:
+            for order_book_row in self.ask_entries():
+                total_cost += order_book_row.amount * order_book_row.price
+                total_volume += order_book_row.amount
+                if total_cost >= quote_volume:
+                    incremental_base_asset = (total_cost-quote_volume)/order_book_row.price
```

Comment:
are you sure you have flake8 setup? it should warn for no spaces between operator.

our standard is to have `a / b` not `a/b`

Reply:
Look at my comment before, I already did ./install. I will change this.